doc/user/guide.rst

```diff
@@ -199,16 +161,16 @@ you should load in paired end reads, or longer reads, first.
 Iterative and independent normalization
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can use :option:`--loadtable` and :option:`--savetable` to do iterative
+You can use :option:`--loadgraph` and :option:`--savegraph` to do iterative
 normalizations on multiple files in multiple steps. For example, break ::
 
   normalize-by-median.py [ ... ] file1.fa file2.fa file3.fa
 
 into multiple steps like so::
 
-  normalize-by-median.py [ ... ] --savetable file1.ct file1.fa
-  normalize-by-median.py [ ... ] --loadtable file1.ct --savetable file2.ct file2.fa
-  normalize-by-median.py [ ... ] --loadtable file2.ct --savetable file3.ct file3.fa
+  normalize-by-median.py [ ... ] --savegraph file1.ct file1.fa
```

Comment:
so, turns out, in the codebase this is "savetable" and it's in a few hundred locations.

Are we gonna change all of the stuff in the codebase to loadgraph?